tensorflow-serving/0000a-tensorflow_patches.patch

```diff
@@ -310,21 +244,20 @@ index 00000000..b4ad792a
 +     deps = [
 +         ":tensorrt_headers",
 +diff --git a/third_party/tensorrt/tensorrt_configure.bzl b/third_party/tensorrt/tensorrt_configure.bzl
-+index 9c980a92cf8..deafd2a5a8a 100644
++index 9c980a9..fe1830f 100644
 +--- a/third_party/tensorrt/tensorrt_configure.bzl
 ++++ b/third_party/tensorrt/tensorrt_configure.bzl
-+@@ -101,6 +101,8 @@ def _create_local_tensorrt_repository(repository_ctx):
++@@ -101,6 +101,7 @@ def _create_local_tensorrt_repository(repository_ctx):
 + 
 +     # Copy the library and header files.
 +     libraries = [lib_name(lib, cpu_value, trt_version) for lib in _TF_TENSORRT_LIBS]
-++    libraries.append("libmemcpy-2.14.so")
```

Comment:
If for 7.0, we are still using libmemcpy.so from TRT package, then this line would be needed.

Reply:
This line is in patch `b` now which does get pulled in for py36 and py37 and cuda 10.2